FirebaseAdmin/FirebaseAdmin.Tests/Auth/ImportUserRecordArgsTest.cs

```diff
@@ -0,0 +1,132 @@
+// Copyright 2020, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace FirebaseAdmin.Auth.Tests
+{
+    public class ImportUserRecordArgsTest
+    {
+        [Fact]
+        public void TestImportUserRecordArgsSerializationBasic()
```

Comment:
We don't use the `Test*` prefix for .NET test case methods. This can be called `SerializationBasic()`.